gci/views.py

```diff
@@ -1,7 +1,9 @@
 from django.http import HttpResponse
 from datetime import datetime
 from calendar import timegm
+from trav import Travis
 import requests
+import os
```

Comment:
whaa ... pep8 ... `os` belongs above third party libraries.

the current imports are a mess. we'll start to fix them without being disruptive.

could you put `os` under `calendar`, followed by a blank line, then `requests` then `trav`

Reply:
:+1:

Reply:
Wait we don't even need `os`... 1 sec